rclpy/rclpy/qos.py

```diff
@@ -37,9 +38,19 @@ class QoSProfile:
     def __init__(self, **kwargs):
         assert all('_' + key in self.__slots__ for key in kwargs.keys()), \
             'Invalid arguments passed to constructor: %r' % kwargs.keys()
+        if 'history' not in kwargs:
```

Comment:
I think it would be good if this class `QoSProfile` could be created with just a `depth`, which this case doesn't allow for.

Reply:
Done 1a47d59